### PR TITLE
Update cloud-provider-openstack maintainers

### PR DESCRIPTION
**Pull request by mdbooth**
Update to reflect changes in https://github.com/kubernetes/cloud-provider-openstack/pull/2318

- [ ] https://github.com/kubernetes/community/pull/7466

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -189,9 +189,10 @@ aliases:
   provider-ibmcloud:
     - spzala
   provider-openstack:
-    - chrigl
-    - lingxiankong
-    - ramineni
+    - dulek
+    - jichenjc
+    - kayrus
+    - zetaab
```
Comment (nikhita) [inline OWNERS_ALIASES:195]: This should reflect - https://github.com/kubernetes/community/blob/9fa912b7e12704d4f1e060ee21f1ccfe53a46315/OWNERS_ALIASES#L183-L186

Please update the OWNERS_ALIASES in k/community first. Once k/community is updated, we can merge this PR.

/hold

Comment (mdbooth) [reply OWNERS_ALIASES:195]: Bah, missed one. Thanks!

Incidentally, I don't supposed there any automation around this I'm missing? Finding all the places is hard and error prone.

Comment (mdbooth) [reply OWNERS_ALIASES:195]: @nikhita I've created a PR: https://github.com/kubernetes/community/pull/7466

Comment (nikhita) [reply OWNERS_ALIASES:195]: > Incidentally, I don't supposed there any automation around this I'm missing? Finding all the places is hard and error prone.

No automation yet 😞 
but this is a good idea. cc @MadhavJivrajani @Priyankasaggu11929 (sig contribex TLs)

Comment (mdbooth) [reply OWNERS_ALIASES:195]: @nikhita The above merged. Can we bump this one along again?

**Comment by mdbooth** [comment]
@dims Any chance you could take a look?

**Comment by mrbobbytables** [comment]
/approve
Sorry, this fell through the cracks for me.

**Comment by zetaab** [comment]
@nikhita @mdbooth we have labels but hold still exists. Can we release it or what is the situation?

**Comment by mrbobbytables** [comment]
The hold can be removed, but we are having some issues with our job that syncs all the orgs. If you don't mind please hold off a little big longer let us get that sorted. 🙏

**Comment by mrbobbytables** [comment]
hm..actually, we can merge this to test, but you may not get added to the group if the job is still having issues. 
/hold cancel

**Comment by mrbobbytables** [comment]
The job completed \o/

**Pull request merged**